srcpkgs/ghc/template

```diff
@@ -39,6 +39,19 @@ desc_option_bindist="Create a binary distribution"
 # GHC's bundled tarball is from 2017, buggy on some archs
 _ffi_rev=4d6d2866ae43e55325e8ee96561221804602cd7a
 
+case "$XBPS_TARGET_MACHINE" in
+aarch64)
+        hostmakedepends+=" llvm"
+        depends+=" llvm"
+        ;;
+aarch64-musl)
+        # does not work with gold
+        configure_args+=" --disable-ld-override"
```

Comment:
This should probably be added for all archs unconditionally.

Reply:
Gold is supposed to be faster, so if it works on other architectures, why disable it?

Reply:
While faster, the code base bitrots a bit more with every binutils release and has multiple issues with musl/archs other than x86_64.

We have disabled gold almost everywhere, including chromium.